pkg/gitprovider/gitprovider.go

```diff
@@ -118,6 +118,17 @@ type PullRequest struct {
 	HeadSHA string `json:"headSHA"`
 	// CreatedAt is the time the pull request was created.
 	CreatedAt *time.Time `json:"createdAt"`
+	// Mergeable indicates whether the underlying hosting provider considers
+	// the PR mergeable (checks passed, no conflicts, etc.). A nil value
+	// indicates the provider could not determine mergeability.
+	Mergeable *bool `json:"mergeable,omitempty"`
+	// Draft indicates whether the PR is a draft (true) or not (false).
+	Draft *bool `json:"draft,omitempty"`
```

Comment:
Mergeable makes sense why it is a pointer, but I don't think this one needs to be a pointer as it will always be either true or false